tox.ini

```diff
@@ -53,14 +53,32 @@ commands =
   # Saves a report to coverage.xml (uploaded to Codecov from CI, as Codecov apparently does not accept .coverage binary files)
   coverage xml
 
-[testenv:docs]
-description = "Build documentation"
+[testenv:build_sphinx_docs]
+description = "Build sphinx documentation"
 usedevelop=true
 deps =
-  -r{toxinidir}/requirements.docs.txt
+  -r{toxinidir}/requirements.sphinx_docs.txt
 commands =
   sphinx-build -W --keep-going -b html docs/source docs/html
 
+[testenv:build_cmd_docs]
+description = "Build command line documentation"
+usedevelop=true
+deps =
+  -r{toxinidir}/requirements.cmd_docs.txt
+commands =
+  python docs/generate_docs.py
+
+
+[testenv:check_docs]
+description = "Check if command line documentation is up to date with sphinx documentation"
+usedevelop=true
+allowlist_externals = /usr/local/bin/bash
```

Comment:
Hmm `bash` typically lives under `/usr/bin/bash`; `/usr/local/bin/` is for stuff that doesn't come from official package repos: https://unix.stackexchange.com/a/4187

Reply:
@PawelLipski I tried using `usr/bin/bash` but got this warning: 
```
WARNING: test command found but not installed in testenv
  cmd: /usr/local/bin/bash
  env: /Users/amalota/Projects/git-machete/.tox/check-docs
Maybe you forgot to specify a dependency? See also the allowlist_externals envconfig setting.

DEPRECATION WARNING: this will be an error in tox 4 and above!
```

Reply:
Hmmm is it possible to just add `bash`, without specifying the exact path? As the path might differ b/w systems/machines

Reply:
Oh yeah, i didn't think about it, it is indeed possible.